### ISSUE TITLE
Documentation inconsistencies for variable names

### ISSUE THREAD

**Issue by scottsweb** [tylerchilds/cutestrap#54]
Hi, thanks for the great project. 

I have been looking through the documentation and spotted some variable references like the following example:

```
html {
    --color-primary-tint: hsl(350, 100%, 80%);
    --color-primary: hsl(350, 80%, 60%);
    --color-primary-shade: hsl(350, 60%, 40%);
}
```
Source: https://www.cutestrap.com/features/themes

But after inspecting the code for the project, I can see that many of these variable names differ slightly. In the case of the above, they are suffixed with numbers, e.g. ` --color-primary-tint1`, source: https://github.com/tylerchilds/cutestrap/blob/master/src/css/core.css#L39

I have noticed this in a few other locations too. Does the documentation need updating?

**Comment by tylerchilds** [comment]
Yeah, I'm not sure why I suffixed them with numbers at this point, but good catch on the mismatch :) Documentation should be updated
